scripts/check_java_code_style.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Copyright 2016 Twitter. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DIR=`dirname $0`
+source ${DIR}/detect_os_type.sh
+
+bazel build --config=`platform` heron/spi/src/java:heron-spi --experimental_action_listener=tools/java:compile_java
```

Comment:
I don't think we need a bash script just to effectively do this:
```
bazel build --config=[config] heron/spi/src/java:heron-spi
```

The `experimental_action_listener` is registered for each platform so it should always run whenever building a java target.